src/pages/[platform]/build-a-backend/storage/define-storage/index.mdx

```diff
@@ -0,0 +1,166 @@
+
+</Callout>
+
+By defining storage in your backend configuration, you can customize the access rules to different paths in your storage bucket.
+
+First, create a file `amplify/storage/resource.ts`. Paste the following content into the file.
+
+```ts title="amplify/storage/resource.ts"
+import { defineStorage } from '@aws-amplify/backend';
+
+export const storage = defineStorage({
+  name: 'myProjectFiles'
+});
+```
+
+Then include storage in your backend definition.
+
+```ts title="amplify/backend.ts"
+import { defineBackend } from '@aws-amplify/backend';
+import { auth } from './auth/resource';
+// highlight-next-line
+import { storage } from './storage/resource';
+
+defineBackend({
+  auth,
+  // highlight-next-line
+  storage
+});
+```
+
+Now when you run `npx amplify sandbox` or deploy your app on Amplify, it will configure an S3 bucket where the files will be uploaded to and downloaded from.
+
+Before files can be accessed by your application, you must configure storage access rules.
+
+## Define File Prefix Access
+
+By default, no users or other project resources have access to any files in the storage bucket. Access must be explicitly granted within `defineStorage` using the `access` callback.
+
+```ts title="amplify/storage/resource.ts"
+export const storage = defineStorage({
+  name: 'myProjectFiles',
+  access: (allow) => ({
+    // highlight-next-line
+    'some/path/*': [
+      // access rules that apply to all files within "/some/path/*" go here
+    ],
+    // highlight-next-line
+    'another/path/*': [
+      // access rules that apply to all files within "/another/path/*" go here
+    ]
+  })
+});
+```
+
+The access callback returns an object where each key in the object is a file prefix and each value in the object is a list of access rules that apply to that prefix. 
+
+Refer to the following example to construct access rules for your storage bucket or you can copy paste to test it out. [Learn more about customizing access to file prefixes](/[platform]/build-a-backend/storage/authorization/).
+
+```ts title="amplify/storage/resource.ts"
+export const storage = defineStorage({
+  name: 'myProjectFiles',
+  access: (allow) => ({
+    'authenticated_access_only/*': [
+      // highlight-next-line
+      allow.authenticated.to(['read', 'write', 'delete']),
+    ],
+    'user_protected/{entity_id}/*': [
+      // highlight-next-line
+      allow.authenticated.to(['read']),
+      // highlight-next-line
+      allow.entity('identity').to(['read', 'write', 'delete'])
+    ],
+    'guest_access_only/*': [
+      // highlight-next-line
+      allow.guest.to(['read', 'write', 'delete'])
+    ]
+  })
+});
+```
+Copy paste the following template for authorization rules to implement public, protected, and private prefixes (this is currently the pattern in Gen 1).
```

Comment:
```suggestion
Copy and paste the following definition for authorization rules to implement public, protected, and private prefixes (this is currently the pattern in Gen 1).
```

to align with previous callouts to "definition", others note "configuration" but this can be ambiguous